### ISSUE TITLE
User-agent: * is expected to be accepted lastly

### ISSUE THREAD

**Issue by ranvis** [diggin/Diggin_RobotRules#9]
Even if a record of User-agent: * comes first, I suppose the most matching record is expected to be used solely.
```php
    {

$txt = <<<EOF
User-agent: *
Disallow: /foo/bar/

User-agent: Googlebot
Disallow: /test/
EOF;

        $accepter = new TxtAccepter();
        $accepter->setRules(TxtStringParser::parse($txt));

        $accepter->setUserAgent('Googlebot');
        $this->assertFalse($accepter->isAllow('/test/1.jpg'));
        $this->assertTrue($accepter->isAllow('/foo/bar/baz.html'));

    }
```

**Comment by sasezaki** [comment]
@ranvis fixed with #16 . and tagging 0.10.0 just released. 
Please, try this.

**Comment by ranvis** [comment]
Thank you for implementing this.
I upgraded and so far it worked perfect!
